src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/FreeText.java

```diff
@@ -0,0 +1,53 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+
+/**
+ * Example FTX+RGI+++WRONG HA - TRY SURREY'
+ */
+@RequiredArgsConstructor
+@Getter
+public class FreeText extends Segment {
+    private static final String KEY = "FTX";
+    private static final String QUALIFIER = "RGI";
+    public static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    private final String textLiteral;
+
+    public static FreeText fromString(String edifactString) {
+        if (!edifactString.startsWith(FreeText.KEY_QUALIFIER)) {
+            throw new IllegalArgumentException("Can't create " + FreeText.class.getSimpleName() + " from " + edifactString);
+        }
+        String[] split = edifactString.split("'")[0].split("\\+");
```

Comment:
the `'` is incorrect here - see `ROW_DELIMITER` in `EdifactMessage` class (can by fixed later with `Split` class I'm preparing)

Reply:
you are right but as it's the delimiter, we can safely split by it and take the 0-index element

Reply:
doesn't break anything but if you prefer, I'll remove that

Reply:
There's just possibility that it can be escaped by `?` - I'll fix that using `Split` class